build.sbt

```
@@ -21,26 +35,28 @@ lazy val codegen = (project in file("codegen"))
     name := "twinagle-scalapb-plugin",
     addSbtPlugin("com.thesamet" % "sbt-protoc" % "1.0.7"),
     libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % scalapb.compiler.Version.scalapbVersion,
-    buildInfoKeys                                 := Seq[BuildInfoKey](version, scalaBinaryVersion),
-    buildInfoPackage                              := "com.soundcloud.twinagle.codegen",
-    buildInfoUsePackageAsPath                     := true,
-    publishLocal                                  := publishLocal.dependsOn(runtime / publishLocal).value,
+    buildInfoKeys := Seq[BuildInfoKey](version, scalaBinaryVersion),
+    buildInfoPackage := "com.soundcloud.twinagle.codegen",
+    buildInfoUsePackageAsPath := true,
+    publishLocal := publishLocal.dependsOn(runtime / publishLocal).value,
     scriptedLaunchOpts ++= Seq("-Xmx1024M", "-Dplugin.version=" + version.value),
     scriptedBufferLog := false
   )
 
 lazy val runtime = (project in file("runtime")).settings(
   commonSettings,
-  name               := "twinagle-runtime",
-  crossScalaVersions := Seq(scala212, scala213),
-  libraryDependencies ++= Seq(
-    "com.twitter"          %% "finagle-http"    % "21.12.0",
-    "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion,
-    "com.thesamet.scalapb" %% "scalapb-json4s"  % "0.12.1",
-    "org.json4s"           %% "json4s-native"   % "4.0.7",
-    "org.specs2"           %% "specs2-core"     % "4.20.5" % Test,
-    "org.specs2"           %% "specs2-mock"     % "4.20.5" % Test
-  ),
+  name := "twinagle-runtime",
+  crossScalaVersions := Seq(scala212, scala213, scala213dot11, scala3),
+  libraryDependencies ++= {
+    Seq(
+      "com.twitter" %% "finagle-http" % "24.2.0" cross CrossVersion.for3Use2_13,
+      "com.thesamet.scalapb" %% "scalapb-runtime" % scalapb.compiler.Version.scalapbVersion cross CrossVersion.for3Use2_13,
+      "com.thesamet.scalapb" %% "scalapb-json4s" % "0.12.1" cross CrossVersion.for3Use2_13,
+      "org.json4s" %% "json4s-native" % "4.1.0-M8" cross CrossVersion.for3Use2_13,
+      "org.specs2" %% "specs2-core" % "4.20.5" cross CrossVersion.for3Use2_13,
```

this is a `Test` dependency:
```suggestion
      "org.specs2" %% "specs2-core" % "4.20.5" % Test cross CrossVersion.for3Use2_13,
```